includes/class-wc-stripe-account.php

```
@@ -472,6 +472,9 @@ public function is_webhook_enabled() {
 		} catch ( Exception $e ) {
 			WC_Stripe_Logger::error( 'Unable to determine webhook status', [ 'error_message' => $e->getMessage() ] );
 			return false;
+		} finally {
+			// Ensure the secret key is reset to ensure non-webhook API calls use the correct key.
+			WC_Stripe_API::set_secret_key( '' );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_API' in method 'is_webhook_enabled'. (undefined)

(StaticAccess)